tests/ttnn/unit_tests/operations/test_small_resnet50_block.py

```diff
@@ -118,6 +118,8 @@ def __call__(self, x, device, batch_size, input_height, input_width, conv_op_cac
             conv_config=ttnn.Conv2dConfig(
                 dtype=self.model_config["ACTIVATIONS_DTYPE"],
                 weights_dtype=self.model_config["WEIGHTS_DTYPE"],
+            ),
+            compute_config=ttnn.GetComputeKernelConfig(
```

Comment:
This will change math fidelity compared to the original code as defaults are not the same